docs/user/openstack/install_upi.md

```diff
@@ -269,6 +269,19 @@ api.openshift.example.com.    A 203.0.113.23
 
 They will need to be available to your developers, end users as well as the OpenShift installer process later in this guide.
 
+## Dual-stack network and Ports (Required for dualstack)
+
+First, revisit all values in the inventory files `dualstack-inventory.yaml` and `inventory.yaml`.
+
+To create a dual-stack cluster, the cluster Network, Subnets and VIPs ports must be pre-created. Please note, that the `dualstack-inventory.yaml` and `inventory.yaml` files will have the values for the API and Ingress VIPs overwritten. Run the following playbook to create necessary resources:
+
+```sh
+$ ansible-playbook -i inventory.yaml -i dualstack-inventory.yaml dualstack-network.yaml
```

Comment:
If we treat this as BYON, do you think we should attempt to delete these resources?

Reply:
Yes, because we create these resources (they're not pre-existing), and these playbooks are just example that users are expected to adapt to their environment.